packages/hdwallet-ledger/src/bitcoin.ts

```diff
@@ -146,9 +146,7 @@ export async function btcSignTx (wallet: BTCWallet, transport: LedgerTransport,
     let path = addressNListToBIP32(msg.inputs[i].addressNList)
     let vout = msg.inputs[i].vout
 
-    let tx = await transport.call('Btc', 'splitTransaction', msg.inputs[i].hex,
-      networksUtil[slip44].isSegwitSupported,
-      networksUtil[slip44].areTransactionTimestamped)
+    let tx = await transport.call('Btc', 'splitTransaction', msg.inputs[i].hex, networksUtil[slip44].isSegwitSupported, networksUtil[slip44].areTransactionTimestamped)
```

Comment:
split by lines easier to read ? (subjective)

Reply:
Going to leave them for now. We can get a linter at some point to make those decisions, but I don't want to revert the change atm.